pkgs/development/python-modules/pwntools/default.nix

```diff
@@ -58,7 +53,19 @@ buildPythonPackage rec {
     intervaltree
   ];
 
-  doCheck = false; # no setuptools tests for the package
+  # There are no setuptools tests
+  doCheck = false;
+
+  # Upstream has set an upper bound on unicorn because of https://github.com/Gallopsled/pwntools/issues/1538,
+  # but since that is a niche use case and it requires extra work to get unicorn 1.0.2rc3 to work we relax
+  # the bound here. Check if this is still necessary when updating!
+  postPatch = ''
+    sed -i 's/unicorn>=1.0.2rc1,<1.0.2rc4/unicorn>=1.0.2rc1/' setup.py
+  '';
```

Comment:
Why did you move this?

Reply:
I have (almost) no preferences for how stuff is ordered in a derivation, but when you asked me to move the other one I thought it was more logical to group the phases. If you tell me how you want it then I'll do that.

Reply:
Patch stuff first, then build stuff, then test install stuff, then check stuff. That's the order in which they are executed.